components/com_joomgallery/models/image.php

```diff
@@ -325,33 +388,14 @@ public function includeWatermark($file, $src_img = null, $cropwidth = 0, $crophe
     }
 
     // Watermark procedure
-    switch($info_wat[2])
-    {
-      case 1:
-        $watermark  = imagecreatefromgif($watermark);
-        $mime_wat   = 'image/gif';
-        break;
-      case 2:
-        $watermark  = imagecreatefromjpeg($watermark);
-        $mime_wat   = 'image/jpeg';
-        break;
-      case 3:
-        $watermark  = imagecreatefrompng($watermark);
-        $mime_wat   = 'image/png';
-        break;
-     default:
-          $this->setError(JText::sprintf('COM_JOOMGALLERY_COMMON_MSG_MIME_NOT_ALLOWED', $info_wat[2]));
-
-        return false;
-    }
 
-    $watermark_width  = imagesx($watermark);
-    $watermark_height = imagesy($watermark);
+    $newwatermark_width  = imagesx($newwatermark);
+    $newwatermark_height = imagesy($newwatermark);
 
     imagealphablending($src_img, true);
-    imagealphablending($watermark, true);
```

Comment:
Wenn ich in Zeile 323 imagealphablending($newwatermark, true); verwende, wird das Wasserzeichen nicht mehr transparent dargestellt. Warum das so ist kann ich dir auch nicht erklären. Wenn ich die Doku auf php.net richtig verstehe müsste es mit true funktionieren. Tut es aber irgendwie nicht.